pkg/telnet/server.go

```diff
@@ -114,3 +118,14 @@ func (ts *Server) Accept() (*Conn, error) {
 	go tc.startNegotiation()
 	return tc, nil
 }
+
+// Close call close on the underlying net.Listener and ensures
+// go routines are stopped
+func (ts *Server) Close() error {
+	err := ts.ln.Close()
+
+	// so far I have not identified any routines that need to be
+	// explicitly stopped after the underlying connection is closed.
```

Comment:
Is there a symptom someone might observe that would indicate that a routine would need to be explicitly stopped here? If so, can we document that?

Reply:
We can add basic doc note:
```
// if vspc server fails to shut down in a timely manner then the clean endpointVM shutdown will fail
// this failure to shut down cleanly can be observed in the "/var/log/vic/init.log" or
// "[datastore] endpointVM/tether.debug" log files as vic-init will report that it's waiting for the portlayer to exit.
```